src/mlpack/tests/cv_test.cpp

```diff
@@ -113,7 +113,7 @@ void CheckPredictionsType()
 BOOST_AUTO_TEST_CASE(PredictionsTypeTest)
 {
   CheckPredictionsType<LinearRegression, arma::rowvec>();
-  CheckPredictionsType<FFN<>, arma::mat>();
```

Comment:
I have disabled the check since API for FFN have changed (now the Train method takes data and predictions as values, rather than as const references).